tools/functional-tester/etcd-tester/tester.go

```diff
@@ -262,12 +256,12 @@ func (tt *tester) cleanup() error {
 
 	plog.Printf("%s cleaning up...", tt.logPrefix())
 	if err := tt.cluster.Cleanup(); err != nil {
-		plog.Printf("%s cleanup error: %v", tt.logPrefix(), err)
+		plog.Warningf("%s cleanup error: %v", tt.logPrefix(), err)
```

Comment:
@heyitsanthony `cleanup` logs the error before it returns, so it would be duplicate if we do that?